spec/fortnox/api/repositories/invoice_spec.rb

```diff
@@ -131,15 +131,18 @@ def new_invoice(comments:)
       before { persisted_invoice }
 
       context 'when setting value to nil' do
-        subject { updated_persisted_invoice.comments }
+        subject(:comments) { updated_persisted_invoice.comments }
 
         let(:updated_persisted_invoice) do
           VCR.use_cassette("#{vcr_dir}/save_old_with_nil_comments") do
             repository.save(persisted_invoice.update(comments: nil))
           end
         end
 
-        pending { is_expected.to eq(nil) }
+        it do
+          pending "test to rerecord VCR cassette, maybe it's working now"
```

Comment:
This shouldn't still be in here, right? Sounds like leftover debug 😄

Reply:
It should still be there. I'll have a look when I rerecord the cassettes.